warehouse/oidc/forms/__init__.py

```diff
@@ -9,18 +9,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import wtforms
 
-from warehouse import forms
-from warehouse.i18n import localize as _
+from warehouse.oidc.forms._core import DeletePublisherForm
+from warehouse.oidc.forms.github import GitHubPublisherForm, PendingGitHubPublisherForm
 
-
-class DeletePublisherForm(forms.Form):
-    __params__ = ["publisher_id"]
-
-    publisher_id = wtforms.StringField(
-        validators=[
-            wtforms.validators.InputRequired(message=_("Specify a publisher ID")),
-            wtforms.validators.UUID(message=_("Publisher must be specified by ID")),
-        ]
-    )
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏nice move - I've started to really like `__init__.py` files that do nothing other than expose other imports.